test/testapps/BasicTestApp/wwwroot/index.html

```diff
@@ -8,32 +8,34 @@
-      <option value="BasicTestApp.ComponentRefComponent">Component ref component</option>
-      <option value="BasicTestApp.AfterRenderInteropComponent">After-render interop component</option>
-      <!--<option value="BasicTestApp.RouterTest.Default">Router</option> Excluded because it requires additional setup to work correctly when loaded manually -->
-    </select>
+      <select onchange="mountTestComponent(event.target.value)">
+          <option value="">Choose...</option>
+          <option value="BasicTestApp.AsyncEventHandlerComponent">Async event handlers</option>
+          <option value="BasicTestApp.AddRemoveChildComponents">Add/remove child components</option>
+          <option value="BasicTestApp.CounterComponent">Counter</option>
+          <option value="BasicTestApp.CounterComponentUsingChild">Counter using child component</option>
+          <option value="BasicTestApp.CounterComponentWrapper">Counter wrapped in parent</option>
+          <option value="BasicTestApp.FocusEventComponent">Focus events</option>
+          <option value="BasicTestApp.KeyPressEventComponent">Key press event</option>
+          <option value="BasicTestApp.MouseEventComponent">Mouse events</option>
+          <option value="BasicTestApp.ParentChildComponent">Parent component with child</option>
+          <option value="BasicTestApp.PropertiesChangedHandlerParent">Parent component that changes parameters on child</option>
+          <option value="BasicTestApp.RedTextComponent">Red text</option>
+          <option value="BasicTestApp.RenderFragmentToggler">Render fragment renderer</option>
+          <option value="BasicTestApp.TextOnlyComponent">Plain text</option>
+          <option value="BasicTestApp.HierarchicalImportsTest.Subdir.ComponentUsingImports">Imports statement</option>
+          <option value="BasicTestApp.HttpClientTest.HttpRequestsComponent">HttpClient tester</option>
+          <option value="BasicTestApp.HttpClientTest.CookieCounterComponent">HttpClient cookies</option>
+          <option value="BasicTestApp.BindCasesComponent">@bind cases</option>
+          <option value="BasicTestApp.ExternalContentPackage">External content package</option>
+          <option value="BasicTestApp.SvgComponent">SVG</option>
+          <option value="BasicTestApp.SvgWithChildComponent">SVG with child component</option>
+          <option value="BasicTestApp.LogicalElementInsertionCases">Logical element insertion cases</option>
+          <option value="BasicTestApp.ElementRefComponent">Element ref component</option>
+          <option value="BasicTestApp.ComponentRefComponent">Component ref component</option>
+          <option value="BasicTestApp.AfterRenderInteropComponent">After-render interop component</option>
+          <!--<option value="BasicTestApp.RouterTest.Default">Router</option> Excluded because it requires additional setup to work correctly when loaded manually -->
+      </select>
```

Comment:
Yes, I guess it's time to stop fighting VS over the indentation here!

Reply:
The alternative would be to use an `.editorsettings` - VS should honour that.